### ISSUE TITLE
set min size of prop symbols

### ISSUE THREAD

**Issue by rgdonohue** [kerrygathers/oil-and-bone#3]
One solution for this would be to use D3's scale module. https://github.com/d3/d3-scale

So, rather than simply deriving the radius directly from the value:

```
var radius = Math.sqrt(val / Math.PI);
return radius * .015; // adjust .25 as a scale factor
```

... we can map the min/max values to a min/max of circle sizes we want:

```
var radius = d3.scale.sqrt()
    .domain([min, max])
    .range([8, 40]);
```

**Comment by rgdonohue** [comment]
One solution for this would be to use D3's scale module. https://github.com/d3/d3-scale

So, rather than simply deriving the radius directly from the value:

```
var radius = Math.sqrt(val / Math.PI);
return radius * .015; // adjust .25 as a scale factor
```

... we can map the min/max values to a min/max of circle sizes we want:

```
var radius = d3.scale.sqrt()
    .domain([min, max])
    .range([0, 10]);
```

**Comment by kerrygathers** [comment]
@rgdonohue This seems like the right solution, thanks. I made a branch called d3-scale to test it out. The circles aren't generating at all, but I'm not getting any errors in the console. Can you take a look and let me know what I'm missing?

**Comment by kerrygathers** [comment]
Implemented with #13
